apple/RNSVGUIKit.h

```diff
@@ -0,0 +1,41 @@
+// Most (if not all) of this file could probably go away once react-native-macos's version of RCTUIKit.h makes its way upstream.
+// https://github.com/microsoft/react-native-macos/issues/242
+
+#if !TARGET_OS_OSX
+
+#import <UIKit/UIKit.h>
+
+#define RNSVGColor UIColor
+#define RNSVGPlatformView UIView
+#define RNSVGTextView UILabel
+#define RNSVGView UIView
+
+#else // TARGET_OS_OSX [
+
+#import <React/RCTUIKit.h>
+
+#define RNSVGColor NSColor
+#define RNSVGPlatformView NSView
+#define RNSVGTextView NSTextView
+
+@interface RNSVGView : RCTUIView
+
+@property CGPoint center;
+@property (nonatomic, strong) RNSVGColor *tintColor;
+
+@end
+
+// TODO: These could probably be a part of react-native-macos
```

Comment:
I'm bad at doing this myself but we should try and pair TODO's with logged issues and then add the issue bug number as a comment here

Reply:
Done.